addons/docs/docs/props-tables.md

```diff
@@ -60,7 +60,7 @@ Starting in SB 6.0, the `ArgsTable` block has built-in `Controls` (formerly know
 
 <br/>
 
-These controls are implemented appear automatically in the props table when your story accepts [Storybook Args](https://github.com/storybookjs/storybook/blob/next/docs/src/pages/formats/component-story-format/index.md#args-story-inputs) as its input. This is done slightly differently depending on whether you're using `DocsPage` or `MDX`.
+These controls are implemented appear automatically in the props table when your story accepts [Storybook Args](https://storybook.js.org/docs/formats/component-story-format#args-story-inputs) as its input. This is done slightly differently depending on whether you're using `DocsPage` or `MDX`.
```

Comment:
```suggestion
These controls are implemented appear automatically in the props table when your story accepts [Storybook Args](https://storybook.js.org/docs/react/api/csf#args-story-inputs) as its input. This is done slightly differently depending on whether you're using `DocsPage` or `MDX`.
```

Reply:
I have updated all links and force pushed.